src/plugins/wizard/public/visualizations/vislib/histogram/histogram_vis_type.ts

```diff
@@ -18,7 +18,7 @@ export interface HistogramOptionsDefaults extends BasicOptionsDefaults {
 
 export const createHistogramConfig = (): VisualizationTypeOptions<HistogramOptionsDefaults> => ({
   name: 'histogram',
-  title: 'Histogram',
+  title: 'Bar',
```

Comment:
I did not update the key to `bar` because vislib refers to it as `histogram`. When we update the visualization to use the new library we can fix this there.

Reply:
ah, makes sense, thanks.